deploy.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-pkill python
+ps -ef | grep -i manage.py | awk '{print $2}' | xargs kill
 git pull
 pip install -r requirements.txt
 openssl rsautl -decrypt -inkey $PATH_TO_PRIVATE_KEY_PEM -in myrails/configuration/prod.encrypted -out myrails/configuration/prod.cfg
 python manage.py migrate
-nohup python manage.py runserver 0.0.0.0:8000 &
+nohup /opt/bitnami/python/bin/python manage.py runserver 0.0.0.0:8000 &
```

Comment:
Why are you explicitly writing the path to the Python interpreter? shouldn't you just have to write
`nohup python manage.py ....`

Reply:
i noticed that when running with only the `python` command, it would use a different python interpreter path than the main one, this is just fast solution, in other circumstances, we can change the default python interpreter the OS needs to run when calling `python `command